### ISSUE TITLE
[Spike] Upgrade to latest Typeorm

### ISSUE THREAD

**Issue by hero101** [cherrytwist/Server#4106]
Timebox to half a day

**Comment by hero101** [comment]
@techsmyth @SimoneZaza I suggest for this to be done, since we are 7 versions behind and a year and a half behind on updates.
We faced one issue today that might be fixed in any of the versions.

**Comment by techsmyth** [comment]
@hero101 agreed good to get up to date on typeorm. Also would suggest this is a precursor to the typeorm issue planned for this sprint (usable definitions outside of server repo)

**Comment by valentinyanakiev** [comment]
this won't be done, replaced by drizzle
